kadi/paths.py

```diff
@@ -1,22 +1,70 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 import os
+from pathlib import Path
+
+
+def _version_str(version):
+    """Legacy version string"""
+    return '' if version in (None, 1, '1') else str(version)
 
 
 def SKA_DATA():
-    return os.path.join(os.environ.get('SKA', '/proj/sot/ska'), 'data/kadi')
+    return Path(os.environ.get('SKA', '/proj/sot/ska'), 'data/kadi')
 
 
 def DATA_DIR():
-    return os.path.abspath(os.environ.get('KADI', SKA_DATA()))
+    return Path(os.environ.get('KADI', SKA_DATA())).absolute()
 
 
 def EVENTS_DB_PATH():
-    return os.path.join(DATA_DIR(), 'events3.db3')
+    return DATA_DIR() / 'events3.db3'
+
+
+def IDX_CMDS_PATH(version=None):
+    version = _version_str(version)
+    return DATA_DIR() / f'cmds{version}.h5'
+
+
+def PARS_DICT_PATH(version=None):
+    version = _version_str(version)
+    return DATA_DIR() / f'cmds{version}.pkl'
+
+
+def CMDS_DIR(scenario=None):
+    from kadi.commands import conf
+
+    # Special case for "flight" scenario. This is hardwired to $SKA/data/kadi
```

Comment:
If hardwired and intended for HEAD flight, shouldn't it just be fully hardwired or warn if it isn't?

Reply:
I really struggled with the name and probably have failed to convey the real meaning in the docs, but the `flight` config is really just a proxy for "ONLY use the current $SKA/data/kadi/cmds2.h5 and do not use network resources to get the latest updates to Command Events and/or approved loads". 

So running production load review on HEAD is one good use case, but another one is just to make things faster on a laptop by not bothering with the 5-10 seconds it can take to do the full update (downloading the Google sheet and hitting OCCweb approved products directories). I've done this myself already a few times.

For the most part I think checking for `/proj/sot/ska` is something we should avoid going forward. There may still be cases, but that isn't the best test of being on HEAD.